test/datasets/commonvoice_test.py

```diff
@@ -0,0 +1,80 @@
+import os
+import csv
+import random
+
+from torchaudio.datasets import commonvoice
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+class TestCommonVoice(TempDirMixin, TorchaudioTestCase):
+    backend = 'default'
+
+    root_dir = None
+    data = []
+    _headers = [u"client_ids", u"path", u"sentence", u"up_votes", u"down_votes", u"age", u"gender", u"accent"]
+    _client_ids = [u"0", u"1"]
+    _sentence = [u"test", u"pytorch"]
+    _up_votes = [u"0", u"1"]
+    _down_votes = [u"0", u"1"]
+    _age = [u"0", u"1"]
+    _gender = [u"M", u"W", u"U"]
+    _accent = [u"Y", u"N"]
+    _folder_audio = "clips"
+    sample_rate = 22050
```

Comment:
I checked couple of randomly chosen files and the sample rates are all 48k Hz. Can you confirm?

```
soxi en/clips/common_voice_en_18885791.mp3

Input File     : 'en/clips/common_voice_en_18885791.mp3'
Channels       : 1
Sample Rate    : 48000
Precision      : 16-bit
Duration       : 00:00:03.26 = 156672 samples ~ 244.8 CDDA sectors
File Size      : 26.2k
Bit Rate       : 64.1k
Sample Encoding: MPEG audio (layer I, II or III)
```

Reply:
agreed